kogito-editors-js/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-editors-java</artifactId>
+    <version>8.2.1-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <packaging>jar</packaging>
+  <artifactId>kogito-editors-js</artifactId>
+  <name>Kogito editors JS</name>
+  <description>React/TS/JS-based-components for the Kogito editors</description>
+
+  <build>
+    <plugins>
+
+      <!-- Build kogito-editors-js components -->
+      <plugin>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
+        <configuration>
+          <installDirectory>${project.build.directory}</installDirectory>
+          <nodeVersion>v16.2.0</nodeVersion>
+          <npmVersion>7.15.1</npmVersion>
+          <yarnVersion>v1.22.10</yarnVersion>
```

Comment:
Could we please set these up in parent pom.xml? Would be easier to update in the future.

Reply:
Sure! Thanks for the suggestion @domhanak. I've created this [Jira](https://issues.redhat.com/browse/KOGITO-5466) to not lose track of this enhancement.